plugin/documents.py

```diff
@@ -908,6 +901,18 @@ def _register_async(self) -> None:
                     debug("also registering", listener)
                     listener.on_load_async()
 
+    def _on_view_updated_async(self) -> None:
+        self._code_lenses_debouncer_async.debounce(
+            self._do_code_lenses_async, timeout_ms=self.code_lenses_debounce_time)
+        different, current_region = self._update_stored_region_async()
+        if not different:
+            return
```

Comment:
As has been mentioned in some other issue, it's not really correct to return here if region didn't change as the file might have still changed (especially on file "reload"). The impact of this is very small currently though (pretty much only symbol highlights could misbehave).